apps/wfo-ui-surf/types/types.ts

```diff
@@ -109,20 +109,20 @@ export interface ServiceTicketLog {
 }
 
 export enum ImpactLevel {
-    NO_IMPACT = 'no_impact',
+    NEVER = 'no_impact',
```

Comment:
is this supposed to match?

Reply:
For now I'll leave it like this, but I am planning to discuss it with Wouter or someone from the backend team, because there is some mismatch in the backend